### PR TITLE
Url fixes

### PR DESCRIPTION
**Pull request by oxelson**
First pass at correcting know issues with URLs in links and references.

### CODE REVIEW

File: docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc [hunk 1]
```diff
@@ -37,8 +37,8 @@ image:viewer1.png[Viewer tab]
 Select a variable in the table, right-click to open its context menu, and choose "NCdump Data".
 The "NCDump Variable Data" dialog will pop up. Here you can print (i.e. "dump") its values to the screen.
 
-Here�using link:http://www.adt.unipd.it/corsi/Bianco/www.pcc.qub.ac.uk/tec/courses/f90/stu-notes/F90_notesMIF_5.html#HEADING41[Fortran 90 array section syntax]
-(`start:end:stride`)�you can print (i.e. "dump") all or part of the variable's values to the screen. By default,
+Here�using link:http://www.adt.unipd.it/corsi/Bianco/www.pcc.qub.ac.uk/tec/courses/f90/stu-notes/F90_notesMIF_5.html#HEADING41[Fortran 90 array section syntax]
```
Comment (lesserwhirls) [inline docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc]: Looks like some kind of encoding issue?

Comment (oxelson) [reply docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc]: Could be.  I’ll fix it. Give me a second.

Comment (oxelson) [reply docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc]: Could be.  I’ll fix it. Give me a second.

**Comment by oxelson** [comment]
@lesserwhirls should be good to go.

File: docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc [hunk 2]
```diff
@@ -66,7 +66,7 @@ Note: to write to NetCDF-4, you must have the link:../netcdf4Clibrary.html[C lib
 image:writer1.png[Writer tab]
 
 === NCDump
-The NCDump tab offers functionality similar to�albeit more limited than�the
+The NCDump tab offers functionality similar to�albeit more limited than�the
```
Comment (lesserwhirls) [inline docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc]: Two more and that should have it :-)

Comment (oxelson) [reply docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc]: Lame.  Sorry I missed that.  It's been corrected.

Comment (oxelson) [reply docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc]: @lesserwhirls. Ok, 'tis done.

**Comment by lesserwhirls** [comment]
Thanks @oxelson! I wish text was a thing that just worked :-(

**Pull request merged**